### PR TITLE
Gitlab improvement

### PR DESCRIPTION
**Pull request by shen390s**

### CODE REVIEW

File: pkgs/applications/version-management/gitlab/remove-hardcoded-locations.patch [hunk 1]
```diff
@@ -96,14 +96,31 @@ index e7a8d08..834ecaf 100644
...
+--- a/lib/gitlab/logger.rb
++++ b/lib/gitlab/logger.rb
+@@ -8,21 +8,22 @@
+       build.info(message)
      end
  
-     def format_message(severity, timestamp, progname, msg)
+     def self.read_latest
+-      path = Rails.root.join("log", file_name)
++      path = File.join(ENV['GITLAB_APPLICATION_LOG_PATH'], file_name)
```
Comment (bjornfor) [inline pkgs/applications/version-management/gitlab/remove-hardcoded-locations.patch]: Huh, this looks weird, your change makes GITLAB_APPLICATION_LOG_PATH a path to .../application.log (a file) but then joins it with "file_name"? Doing the "join" sounds more appropriate when GITLAB_APPLICATION_LOG_PATH is a directory path and not a file.

Comment (anderspapitto) [inline pkgs/applications/version-management/gitlab/remove-hardcoded-locations.patch]: If you look above where that variable is defined, this patch actually changes it from a file name to a directory name

**Comment by shen390s** [comment]
Yes. GITLAB_APPLICATION_LOG_PAT has been changed to point to a directory

**Comment by fpletz** [comment]
Sorry, I wasn't aware of this PR when I did #13396. As far as I can see the log path fixes are also in my PR so I'll close this.

**Pull request closed without merging**